src/Kdyby/Validator/Caching/Cache.php

```diff
@@ -56,7 +56,7 @@ public function has($class)
 	 */
 	public function read($class)
 	{
-		return $this->has($class) ? $this->cache[$class] : FALSE;
+		return $this->cache->load($class) ?: FALSE;
```

Comment:
0 is invalid value, see phpdoc of CacheInterface:
`@return ClassMetadata|false A ClassMetadata instance or false on miss`